feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -52,6 +52,7 @@ class PreviewViewModel @Inject constructor(
 
     private fun initializeCamera() {
         // TODO(yasith): Handle CameraUnavailableException
+        Log.d(TAG, "initializeCamera")
```

Comment:
nit: do you want to remove the log statements? ok to leave them in just wondering if they were for debugging purpose

Reply:
Intentionally adding them to help with tracking down future issues.